functions/Set-DbaErrorLog.ps1

```diff
@@ -35,130 +35,98 @@ function Set-DbaErrorLog {
 
             Website: https://dbatools.io
             Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
-            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+            License: MIT https://opensource.org/licenses/MIT
 
         .LINK
             https://dbatools.io/Set-DbaErrorLog
 
        .EXAMPLE
-            Set-DbaErrorLog -SqlInstance server2017,server2014 -NumberOfLog 25
+            Set-DbaErrorLog -SqlInstance sql2017,sql2014 -LogCount 25
 
-            Sets the number of error log files to 25 on server2017 and server2014
+            Sets the number of error log files to 25 on sql2017 and sql2014
 
         .EXAMPLE
-            Set-DbaErrorLog -SqlInstance server2014 -SizeInKb 1024
+            Set-DbaErrorLog -SqlInstance sql2014 -LogSize 102400
 
-            Sets the size of the error log file, before it rolls over, to 1024KB (1GB) on server2014
+            Sets the size of the error log file, before it rolls over, to 102400 KB (100 MB) on sql2014
 
         .EXAMPLE
-            Set-DbaErrorLog -SqlInstance server2012 -NumberOfLog 25 -SizeInKb 500
+            Set-DbaErrorLog -SqlInstance sql2012 -LogCount 25 -LogSize 500
 
-            Sets the number of error log files to 25 and size before it will roll over to 500KB on server2012
+            Sets the number of error log files to 25 and size before it will roll over to 500 KB on sql2012
     #>
     [cmdletbinding(SupportsShouldProcess)]
     param(
-        [Parameter(ValueFromPipelineByPropertyName)]
+        [Parameter(ValueFromPipelineByPropertyName, Mandatory)]
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
-        [DbaValidatePattern('^[6-9][0-9]?$', ErrorMessage = "Error processing {0} - input must be an integer between 6 and 99")]
-        [int]$NumberOfLog,
-        [int]$SizeInKb,
+        [ValidateRange(6, 99)]
+        [int]$LogCount,
+        [int]$LogSize,
         [switch]$EnableException
     )
     process {
         foreach ($instance in $SqlInstance) {
             Write-Message -Level Verbose -Message "Connecting to $instance"
             try {
-                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential -MinimumVersion 11
```

Comment:
You cannot use this.

Reply:
Setting the log file count is supported for any version of SQL Server (2000+). The only thing restricted to 2012+ is the size.